aardwolf-templates/src/widgets/icon_enums.rs

```diff
@@ -0,0 +1,47 @@
+/// Presuming that all ForkAwesome icons follow the same prefix:
+/// `fa fa-[name]
+/// All values of [name] are as below
+/// To add new icons simply find the desired names within ForkAwesome, 
+/// and follow the syntax below:
+/// `IconName::[SnakeCaseIconName] => write!(f, "actual-name"), // Normal icon`
+/// `IconName::[SnakeCaseIconNameOutline] => write!(f, "actual-name-o"), // Outline icon`
+///
+/// Keeping them in alphabetical order would also be nice ;)
+
+
+use std::fmt;
+impl fmt::Display for IconName {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+			IconName::AngleDown => write!(f, "angle-down"),
```

Comment:
these should be un-tabbed one level. try running `cargo fmt` to clean stuff like this. If you don't have cargo fmt, it's just a `rustup component add rustfmt-preview` away

Reply:
ran `cargo fmt`